lib/index.js

```diff
@@ -252,6 +252,27 @@ class VideoAsset extends BaseAsset {
 
     set_download_uri(value) { this._download_uri = value; }
 
+    // Set a wrapper article asset ID on this video. This way,
+    // consumers can know if this article is merely just wrapping
+    // another video asset and can go directly to the video if
+    // appropriate
+    set_wrapper(article) {
+        // The only valid wrapper for VideoAsset is GalleryVideoArticle
+        if (!(article instanceof GalleryVideoArticle)) {
+            throw new Error(`The only valid wrapper for VideoAsset is GalleryVideoArticle`);
```

Comment:
This seems incorrect to me. Third parties should be able to write their own article format for their ingester that inherit from `BaseArticle` if they don't want any of the formats that we provide, so I don't think you can make this restriction.

Reply:
+1, I like this approach as well.

Reply:
Okay. I think in that case I'll make the `instanceof` check operate on `BaseArticle` for now.